obspy/io/nordic/tests/test_nordic.py

```diff
@@ -0,0 +1,698 @@
+                                           resource_id,
+                                           waveform_id=test_event.picks[0].
+                                           waveform_id,
+                                           unit='m',
+                                           magnitude_hint='ML',
+                                           category='point',
+                                           type='AML'))
+    # Test a coda magnitude pick
+    test_event.amplitudes.append(Amplitude(generic_amplitude=10,
+                                           pick_id=test_event.picks[1].
+                                           resource_id,
+                                           waveform_id=test_event.picks[1].
+                                           waveform_id,
+                                           type='END',
+                                           category='duration',
+                                           unit='s',
+                                           magnitude_hint='Mc',
+                                           snr=2.3))
+    test_event.origins[0].arrivals.append(Arrival(time_weight=0,
+                                                  phase=test_event.
+                                                  picks[1].
+                                                  phase_hint,
+                                                  pick_id=test_event.
+                                                  picks[1].
+                                                  resource_id))
+    test_event.origins[0].arrivals.append(Arrival(time_weight=2,
+                                                  phase=test_event.
+                                                  picks[2].
+                                                  phase_hint,
+                                                  pick_id=test_event.
+                                                  picks[2].
+                                                  resource_id,
+                                                  backazimuth_residual=5,
+                                                  time_residual=0.2,
+                                                  distance=15,
+                                                  azimuth=25))
+    test_event.origins[0].arrivals.append(Arrival(time_weight=2,
+                                                  phase=test_event.
+                                                  picks[3].
+                                                  phase_hint,
+                                                  pick_id=test_event.
+                                                  picks[3].
+                                                  resource_id,
+                                                  backazimuth_residual=5,
+                                                  time_residual=0.2,
+                                                  distance=15,
+                                                  azimuth=25))
+
```

Comment:
I wish I had known that was allowed! I will be doing that in future - and here if I get the time. Thanks!